flask_webtest.py

```diff
@@ -8,16 +8,16 @@
 from contextlib import contextmanager
 
 from werkzeug.local import LocalStack
-from flask import g, session, get_flashed_messages
+from flask import g, session, get_flashed_messages, _app_ctx_stack
 from flask.signals import template_rendered, request_started, request_finished
 from webtest import (TestApp as BaseTestApp,
                      TestRequest as BaseTestRequest,
                      TestResponse as BaseTestResponse)
 
 try:
-    from flask_sqlalchemy import connection_stack
+    import flask_sqlalchemy
 except ImportError:
-    connection_stack = None
+    pass
```

Comment:
I think you need to set `flask_sqlalchemy = None` here otherwise when you access it later I think you'll get a `NameError`